app/controllers/cancels_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class CancelsController < ApplicationController
+  include CanFetchResource
+  include FinanceDetailsHelper
```

Comment:
This isn't an issue with the PR. I'm just taking an opportunity to highlight something _I think_ should be refactored and I'm interested in your thoughts.

We've added [FinanceDetailsHelper](https://github.com/DEFRA/waste-carriers-back-office/blob/master/app/helpers/finance_details_helper.rb) to this controller I'm assuming just to access its `details_path_for()` method.

It stands out to me that we are referring to a finance thing, for something that on the face of it doesn't have anything to do with finance.

So should we find ourselves with 5 minutes spare 😁 , I think we should be refactoring `details_path_for()` to a more appropriately named Concern. What do you think?

Reply:
I think we should keep it as a helper because the function is used mostly in views than in controllers. What I suggest is to move the function to a ResourceHelper class at this point. WDYT?

Reply:
If it's used in views more than controllers that makes sense. But don't worry about updating this PR. It should be done in a separate one.